### PR TITLE
Jk html css1

### PR DESCRIPTION
**Pull request by josephkane**
Basic html and css styling

### CODE REVIEW

**Comment by sulaiman-allen** [comment]
:+1:

File: index.html [hunk 1]
```diff
@@ -1,13 +1,71 @@
...
+    </div>
+
+    <div id="battleground" class="row">
+      <div class="player-image col-xs-5">
+
+      </div>
+      <div id="attack-area" class="col-xs-2">
+        <button id="attack-button" type="button">ATTACK!</button>
+      </div>
+      <div class="player-image col-xs-5">
```
Comment (aoswalt) [inline index.html]: Change class to enemy-image for selection

File: index.html [hunk 2]
```diff
@@ -1,13 +1,71 @@
...
+          <li>Name:</li>
+          <li>Species:</li>
+          <li>HP:</li>
+          <li>Weapon:</li>
+        </ul>
+      </div>
+      <div class="col-xs-2">
+      </div>
+      <div class="col-xs-5">
+        <ul class="player-stats">
```
Comment (aoswalt) [inline index.html]: Change class to enemy-stasts for selection

File: index.html [hunk 3]
```diff
@@ -1,13 +1,71 @@
...
+      </div>
+      <div class="player-image col-xs-5">
+
+      </div>
+
+    </div>
+
+    <div id="player-attributes" class="row">
+      <div class="col-xs-5">
+        <ul class="player-stats">
```
Comment (aoswalt) [inline index.html]: Add classes to list items for selection.

File: index.html [hunk 4]
```diff
@@ -1,13 +1,71 @@
...
+        <h1 id="game-headline">GAUNTLET.</h1>
+      </div>
+    </div>
+
+    <div id="player-headlines" class="row">
+      <div id="player-one" class="col-xs-5">
+        <h2>Player 1</h2>
+      </div>
+      <div class="col-xs-2"></div>
+      <div id="player-2" class="col-xs-5">
```
Comment (aoswalt) [inline index.html]: change one of the players to number/not number for consistency

**Comment by aoswalt** [comment]
:+1:

**Pull request merged**